blueprint/core/blueprint_ViewManager.cpp

```diff
@@ -0,0 +1,239 @@
+/*
+  ==============================================================================
+
+    blueprint_ViewManager.cpp
+    Created: 21 May 2020 15:32:00am
+
+  ==============================================================================
+*/
+
+#include "blueprint_ViewManager.h"
+
+namespace blueprint
+{
+    //TODO: Have the root ShadowView owned by ViewManager or have ReactApplicationRoot own this
+    //      and pass a ViewPair like struct to ViewManager constructor?
+    //      Plausible passing a ViewPair fits better with the existing registerViewType
+    //      pattern?
+    ViewManager::ViewManager(View* rootView)
+        : rootShadowView(std::make_unique<ShadowView>(rootView))
```

Comment:
What do you think about the `shadowViewTable` owning this unique_ptr? Maybe in the constructor here we make the root shadow view, put it in the map, and hang onto the `rootViewId` as a member instead of the pointer. That way the methods below that depend on lookups in the shadow view table can also find the root shadow view if they ever need to

Reply:
Yeah I think that's nicer. Agreed. Fill fix up in my next set of changes

Reply:
@nick-thompson, changed made in https://github.com/nick-thompson/blueprint/pull/46/commits/71ed1a32f3c01f094cacf15b242800f33ffd2ce7